fingeringdiagram.qml

```diff
@@ -443,8 +448,10 @@ MuseScore {
 			// Check for supported instrument parts
 			var part = curScore.parts[staff];
 			fingering = new fingeringClass(part);
+			staffFound.push(part.longName);
```

Comment:
i would do this instead to help the user to know the instrument id too: `staffFound.push(part.longName + ' (' + part.instrumentId + ')');`